xchange-kraken/src/main/java/org/knowm/xchange/kraken/dto/account/KrakenEarnAllocation.java

```diff
@@ -0,0 +1,37 @@
+package org.knowm.xchange.kraken.dto.account;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class KrakenEarnAllocation {
+  @JsonProperty("amount_allocated")
+  private final KrakenEarnAmountAllocated amountAllocated;
+
+  @JsonProperty("native_asset")
+  private final String nativeAsset;
+
+  @JsonProperty("payout")
+  private final KrakenEarnPayout payout;
+
+  @JsonProperty("strategy_id")
+  private final String strategyId;
+
+  @JsonProperty("total_rewarded")
+  private final KrakenEarnAmount totalRewarded;
+
+  public KrakenEarnAllocation(
```

Comment:
imho, can be omitted

Reply:
Yes, replaced with @builder + @Jacksonized